build.gradle

```diff
@@ -17,7 +17,7 @@
 
 apply plugin: 'scala'
 
-version = '0.6.3-2'
+version = '0.6.2.3'
```

Comment:
@snappy-sachin Try "git fetch origin" and then "git merge origin/snappydata" into branch-jj-0.6.2.4 again.